### ISSUE TITLE
Connection via socks5 proxy

### ISSUE THREAD

**Issue by FicsrealZ** [ubports/telegram-app#173]
Dear developers,
In Russia the telegram is blocked. The Russian community asks to add the possibility of running the application via socks5 proxy.
Sorry for my bad english.

**Comment by MaxSavenkov** [comment]
I second this request, and also maybe check out some automatic censorship avoidance methods used in the main client? I'm not _quite_ sure how the desktop client does it, but it avoids the block on my PC without even specifying a proxy (but with IPv6 enabled), but the Ubuntu Touch version is blocked on the same connection.

**Comment by Flohack74** [comment]
Yeah we know this. But we will release a complete new App which will use the official tdlib API from Telegram, which will have all those features then. ETA end of year.

**Comment by MaxSavenkov** [comment]
Oh, cool, good to know. Is there a branch for it I can try in a pinch?
